apps/studio/components/interfaces/Organization/NewOrg/NewOrgForm.tsx

```diff
@@ -58,11 +58,11 @@ const NewOrgForm = ({ onPaymentMethodReset }: NewOrgFormProps) => {
   const stripe = useStripe()
   const elements = useElements()
 
-  const { plan } = useParams()
+  const { plan, name, kind, size, spend_cap } = useParams()
 
-  const [orgName, setOrgName] = useState('')
-  const [orgKind, setOrgKind] = useState(ORG_KIND_DEFAULT)
-  const [orgSize, setOrgSize] = useState(ORG_SIZE_DEFAULT)
+  const [orgName, setOrgName] = useState(name || '')
+  const [orgKind, setOrgKind] = useState(kind || ORG_KIND_DEFAULT)
```

Comment:
i just realised - shouldn't `ORG_KIND_DEFAULT` be `ORG_KIND_TYPES.PERSONAL` instead of `"PERSONAL"`?
Below where you do `Object.entries` on `ORG_KIND_TYPES` the values are the sentence cased values instead of the keys